OSX-NOTES.md

```
@@ -48,8 +48,19 @@ Now we need to install some dependencies
+openssl, sqlite, and libtool are pre-installed on the system. The versions
+downloaded by brew are stored in an alternative location, which differs by
+CPU architecture:
+
+  - Intel Macs: /usr/local
+  - Apple Silicon Macs (M1 and later): /opt/homebrew
+
+In addition, recent versions of Homebrew install OpenSSL as a versioned keg
+named openssl@3 rather than plain openssl. You can always find the correct
+path for your machine with:
+
+	$ brew --prefix openssl@3
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)